src/components/RefineSerach.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Text, Button } from '@chakra-ui/react'
+import { useSelector } from 'react-redux';
+import Router, { useRouter } from 'next/router';
+import { RootState } from '../../src/redux/store';
+import Link from 'next/link';
+
+
+
+const RefineSerach = () => {
+    const CategoryValues = useSelector((state: RootState) => state.categories.data);
+    const router = useRouter()
+    const { id } = router.query as { id: string };
+    const Selectcategory = CategoryValues.find((item) => item.id === Number(id));
+    let arrSelectCat = [Selectcategory?.attribute];
+    let itemsOfarrSelect = arrSelectCat[0];
```

Comment:
Why you added them into an array and then picked the first element which is the same value at beginning? 
```suggestion
    let itemsOfarrSelect = Selectcategory?.attribute;
```